src/Server/RestHTTPRequestHandler.cpp

```diff
@@ -402,6 +405,20 @@ RestHTTPRequestHandler::RestHTTPRequestHandler(IServer & server_, const String &
 {
 }
 
+/// FIXME : Get the corresponding database according to the user name
+String RestHTTPRequestHandler::getDatabaseByUser(const String & user) const
+{
+    String database = "default";
+
+    if (!database.empty())
```

Comment:
Hmm, `database.empty()` ? Do we test the latest code ?

Reply:
Sorry, have fixed